flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -148,6 +148,16 @@
 	public static final String TASK_MANAGER_NETWORK_DEFAULT_IO_MODE = "taskmanager.network.defaultIOMode";
 
 	/**
+	 * Initial backoff (in ms) for partition requests, which return partition not found.
+	 */
+	public static final String TASK_MANAGER_NETWORK_REQUEST_BACKOFF_INITIAL_MS = "taskmanager.network.requestBackoffInitial";
```

Comment:
Actually no. I think it's safe to have it fixed. We only need to adjust the values for tests. What is the easiest way to only allow internal configuration? The problem is that for integration tests it's hard to set configuration values for runtime components otherwise.